fractal_server/app/runner/_slurm/executor.py

```diff
@@ -0,0 +1,228 @@
+# This adapts clusterfutures <https://github.com/sampsyo/clusterfutures>
+# Original Copyright
+# Copyright 2021 Adrian Sampson <asampson@cs.washington.edu>
+# License: MIT
+#
+# Copyright 2022 Jacopo Nepsolo <jacopo.nespolo@exact-lab.it>
+import shlex
+import subprocess  # nosec
+import sys
+from concurrent import futures
+from pathlib import Path
+from typing import Any
+from typing import Callable
+from typing import List
+from typing import Optional
+
+import cloudpickle
+from cfut import RemoteException
+from cfut import SlurmExecutor  # type: ignore
+from cfut.util import random_string
+
+from ....config import get_settings
+from ....syringe import Inject
+from ....utils import set_logger
+
+
+def get_slurm_script_dir() -> Path:
+    settings = Inject(get_settings)
+    script_dir = settings.RUNNER_ROOT_DIR  # type: ignore
+    return script_dir  # type: ignore
+
+
+def get_stdout_filename(arg: str = "%j") -> Path:
```

Comment:
See #284.
(this can be done later, it's FYI)

Reply:
will solve #284 separately